HW_1/Task_6.py

```diff
@@ -0,0 +1,15 @@
+num = str(input('enter a six digit number: '))
```

Comment:
Функция ввода данных и уже возвращает строковый тип данных.
```suggestion
num = input('enter a six digit number: ')
```